src/main/webapp/app/overview/course-lectures/online-unit/online-unit.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="row align-items-center mb-2 mt-2">
+    <div class="card unit-card">
+        <div class="card-header unit-card-header row align-content-center" (click)="handleCollapse($event)">
+            <h5 class="m-0 fw-medium">
+                <fa-icon class="col-auto me-2" [icon]="faLink" [ngbTooltip]="'artemisApp.onlineUnit.tooltip' | artemisTranslate"> </fa-icon>
+                {{ onlineUnit?.name ? onlineUnit.name : '' }}
```

Comment:
```suggestion
                {{ onlineUnit?.name ?? '' }}
```
Not 100% sure if this works as well but maybe it is a considerable alternative